kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/ResourceQuotaTest.java

```diff
@@ -0,0 +1,169 @@
+
+    KubernetesClient client = server.getClient();
+
+    Boolean deleted = client.resourceQuotas().withName("resourcequota1").delete();
+    assertTrue(deleted);
+
+    deleted = client.resourceQuotas().withName("resourcequota2").delete();
+    assertFalse(deleted);
+
+    deleted = client.resourceQuotas().inNamespace("ns1").withName("resourcequota2").delete();
+    assertTrue(deleted);
+  }
+
+  @Test
+  public void testeDeleteMulti() {
+    ResourceQuota resourcequota1 = new ResourceQuotaBuilder().withNewMetadata().withName("resourcequota1").withNamespace("test").endMetadata().build();
+    ResourceQuota resourcequota2 = new ResourceQuotaBuilder().withNewMetadata().withName("resourcequota2").withNamespace("ns1").endMetadata().build();
+    ResourceQuota resourcequota3 = new ResourceQuotaBuilder().withNewMetadata().withName("resourcequota3").withNamespace("any").endMetadata().build();
+
+    server.expect().withPath("/api/v1/namespaces/test/resourcequotas/resourcequota1").andReturn(200, resourcequota1).once();
+    server.expect().withPath("/api/v1/namespaces/ns1/resourcequotas/resourcequota2").andReturn(200, resourcequota2).once();
+
+    KubernetesClient client = server.getClient();
+
+    Boolean deleted = client.resourceQuotas().inAnyNamespace().delete(resourcequota1, resourcequota2);
+    assertTrue(deleted);
+
+    deleted = client.resourceQuotas().inAnyNamespace().delete(resourcequota3);
+    assertFalse(deleted);
+  }
+
+  @Test
+  public void testLoadFromFile() {
+    KubernetesClient client = server.getClient();
+    ResourceQuota resourceQuota = client.resourceQuotas().load(getClass().getResourceAsStream("/test-resourcequota.yml")).get();
+    assertEquals("compute-quota", resourceQuota.getMetadata().getName());
+  }
+
+  @Test
+  public void testBuild() {
+    server.expect().withPath("/api/v1/namespaces/myspace/resourcequotas/compute-quota").andReturn(200, new ResourceQuotaBuilder()
+      .withNewMetadata().withName("compute-quota").withNamespace("myspace").endMetadata()
+      .withNewSpec().addToHard("pods", new Quantity("2"))
+      .addToHard("requests.cpu", new Quantity("1"))
+      .addToHard("limits.cpu", new Quantity("2")).endSpec().build()).once();
+
+    KubernetesClient client = server.getClient();
+
+    Deployment deployment = client.apps().deployments().load(getClass().getResourceAsStream("/test-resourcequota-deployment.yml")).get();
+    server.expect().withPath("/apis/apps/v1/namespaces/myspace/deployments/deployment").andReturn(200, deployment).once();
+
+    ResourceQuota resourcequota = client.resourceQuotas().inNamespace("myspace").withName("compute-quota").get();
+    assertNotNull(resourcequota);
+
+    deployment = client.apps().deployments().inNamespace("myspace").withName("deployment").get();
+    assertNotNull(deployment);
+  }
+
+  @Test
+  public void testResources() {
+    Config config = new ConfigBuilder().build();
+    try (final KubernetesClient client = new DefaultKubernetesClient(config)) {
```

Comment:
Why are you not mocking server here? We don't have a cluster running during unit tests, however you can do in in integration tests(`kubernetes-itests/` module).

Reply:
Moving to itests/

Reply:
Hmm, moving to itests/